app/resources/brave_generated_resources_bg.xtb

```diff
@@ -351,7 +354,23 @@
         Можете да прочетете <ph name="BEGIN_LINK_BUILD_INSTRUCTIONS"/>инструкции как да изтеглите и изградите за себе си<ph name="END_LINK_BUILD_INSTRUCTIONS"/> конкретния <ph name="BEGIN_LINK_SOURCE_CODE"/>изходен код, използван за създаването на това копие<ph name="END_LINK_SOURCE_CODE"/>.</translation>
 <translation id="4474155171896946103">Запазване на отметки към всички раздели…</translation>
 <translation id="2336228925368920074">Запазване на отметки към всички раздели...</translation>
+<translation id="4235920711083356755">Показване на бутона VPN</translation>
+<translation id="4070612712151038707">Показване на бутона VPN в лентата с инструменти</translation>
 <translation id="7705524343798198388">VPN</translation>
+<translation id="3960619311638643872">Brave VPN</translation>
+<translation id="2345974803401546682">Относно Brave VPN</translation>
+<translation id="4378551569595875038">Установява се връзка...</translation>
+<translation id="3368922792935385530">Установена е връзка</translation>
+<translation id="1536754031901697553">Връзката се прекратява...</translation>
+<translation id="8030169304546394654">Разкачен</translation>
+<translation id="1863950594388384201">Показване на бутона VPN в лентата с инструменти</translation>
+<translation id="5187139277471084437">Скриване на бутона VPN в лентата с инструменти</translation>
```

Comment:
```suggestion
<translation id="5187139277471084437">Скриване на VPN бутона в лентата с инструменти</translation>
```

"Hide the VPN button in the toolbar" rather than "Hide the button VPN in the toolbar".